app/models/enterprise.rb

```diff
@@ -198,10 +188,6 @@ def self.distinct_count
     count(distinct: true)
   end
 
-  def activated?
-    confirmed_at.present? && sells != 'unspecified'
```

Comment:
I think we still need to concept of `activated`? It behaves like a check on the enterprise to determine whether the registration process is complete, if anything I think this should just become:

````ruby
owner.confirmed? && sells != 'unspecified'
````